### PR TITLE
[Common] Static analyzed issues.

### PR DESCRIPTION
**Pull request by nshchego**

### CODE REVIEW

File: inference-engine/src/readers/ir_reader/ie_ir_reader.cpp [hunk 1]
```diff
@@ -55,7 +55,7 @@ INFERENCE_PLUGIN_API(StatusCode) InferenceEngine::CreateReader(IReader*& reader,
         reader = new IRReader();
         return OK;
     }
-    catch (std::exception &ex) {
+    catch (...) {
```
Comment (ilyachur) [inline inference-engine/src/readers/ir_reader/ie_ir_reader.cpp:58]: Why did you change `std::exception` to any exception?

Comment (nshchego) [reply inference-engine/src/readers/ir_reader/ie_ir_reader.cpp:58]: 'ex' is never used. Code: LV_UNUSED.GEN

Comment (ilya-lavrenov) [reply inference-engine/src/readers/ir_reader/ie_ir_reader.cpp:58]: can we have `catch (std::exception &)` ? Without actual variable, but type of exception is still the same.

File: inference-engine/src/readers/onnx_reader/ie_onnx_reader.cpp [hunk 2]
```diff
@@ -30,7 +30,7 @@ INFERENCE_PLUGIN_API(StatusCode) InferenceEngine::CreateReader(IReader*& reader,
         reader = new ONNXReader();
         return OK;
     }
-    catch (std::exception &ex) {
+    catch (...) {
```
Comment (ilya-lavrenov) [inline inference-engine/src/readers/onnx_reader/ie_onnx_reader.cpp:33]: the same

File: inference-engine/src/legacy_api/src/cnn_network_int8_normalizer.cpp [hunk 3]
```diff
@@ -1003,12 +1003,14 @@ void CNNNetworkInt8Normalizer::QuantizeConvolutionOrFullyConnected(CNNLayer::Ptr
...
-            for (auto it = individualsG.begin(); it != individualsG.end(); it++) {
-                if (prev) {
-                    divs.insert((*it - prev) / symQuant);
+            if (symQuant != 0.) {
+                prev = 0.f;
+                for (auto it = individualsG.begin(); it != individualsG.end(); it++) {
+                    if (prev) {
+                        divs.insert((*it - prev) / symQuant);
+                    }
+                    prev = *it;
```
Comment (gladilov-gleb) [inline inference-engine/src/legacy_api/src/cnn_network_int8_normalizer.cpp:1012]: Nitpick: for-each loop is applicable here?

```cpp
for (const auto& individualG : individualsG) {
    // the same body except renaming it to individualG
}
```

Comment (nshchego) [reply inference-engine/src/legacy_api/src/cnn_network_int8_normalizer.cpp:1012]: Here is zero division protection.

File: inference-engine/src/readers/ir_reader/ie_ir_reader.cpp (hunk 1 above)
Comment (nshchego) [reply inference-engine/src/readers/ir_reader/ie_ir_reader.cpp]: fixed

File: inference-engine/src/readers/onnx_reader/ie_onnx_reader.cpp (hunk 2 above)
Comment (nshchego) [reply inference-engine/src/readers/onnx_reader/ie_onnx_reader.cpp]: fixed

File: inference-engine/src/legacy_api/src/cnn_network_int8_normalizer.cpp (hunk 3 above)
Comment (gladilov-gleb) [reply inference-engine/src/legacy_api/src/cnn_network_int8_normalizer.cpp:1012]: Sure, but my question is not about loop body, but about `for` statement itself. I don't see a reason so far to deal with iterators here.
